std_msgs/Header header
int16 IDLE = 0
int16 TAKEOFF = 1
int16 FLIGHT_LOCAL = 2
int16 FLIGHT_GLOBAL = 3
int16 UNKNOWN = 10
int16 state